### PR TITLE
DOCS-5708: CSM CIEM

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do?
Adds docs for CSM CIEM.

### Motivation
DOCS-5708

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by aliciascott — APPROVED** [review]
Looks good and approved, I just had a couple small edits.

File: content/en/security/cloud_security_management/_index.md [hunk 1]
```diff
@@ -54,6 +53,12 @@ Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#c
 
 {{< img src="security/cws_overview.png" alt="Cloud Workload Security coverage views in Datadog" width="100%">}}
 
+## CSM Identity Risks
+
+[CSM Identity Risks][4] provides in-depth visibility into your organization's AWS IAM risks and helps you to resolve them on an ongoing basis. Enabling you to detect and resolve identity risks on an ongoing basis, CSM Identity Risks helps secure your cloud infrastructure from IAM-based attacks.
```
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/security/cloud_security_management/_index.md:58]: Good catch. I completely missed the fact that I was repeating the same information. I'm going to update the intro and remove the redundant sentence.

File: content/en/security/identity_risks/_index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+<div class="alert alert-warning">
+CSM Identity Risks is not available in this site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">CSM Identity Risks is in beta.</div>
+
+Cloud Security Management Identity Risks (CSM Identity Risks) provides in-depth visibility into your organization's AWS IAM risks. Enabling you to detect and resolve identity risks on an ongoing basis, CSM Identity Risks helps secure your cloud infrastructure from IAM-based attacks.
+
+<div class="alert alert-info">At this time, CSM Identity Risks is available for AWS only.</div>
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:20]: At this time, Cloud Infrastructure Entitlement Management (CIEM) is only available for AWS.

File: content/en/security/identity_risks/_index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Setup
+
+To use CSM Identity Risks, you must [enable resource collection for AWS][2] and [enable CloudTrail logs forwarding][4]. If you've already done this, no additional setup is required.
+
+## Review and remediate identity risk detections
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:26]: Review and remediate identity risks

File: content/en/security/identity_risks/_index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Review your organization's active identity risk detections on the [Identity Risks Explorer page][3]. Use the **Group by** options to filter by **Identity Risks**, **Resources**, or **None** (individual detections). View additional details on the side panel by selecting a resource or detection.
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:28]: Review your organization's identity risks on the [Identity Risks][3] explorer. Use the **Group by** options to organize by **Identity Risks**, **Resources**, or **None** (individual detections). View additional details on the side panel by selecting an identity risk or a resource.

File: content/en/security/identity_risks/_index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+{{< img src="security/identity_risks/identity_risks_explorer.png" alt="CSM Identity Risks explorers page" width="100%">}}
+
+When viewing an individual resource, use the **Relationships** tab to view a graphical representation of the connections with other resources. The view includes both benign and at-risk connections and shows how a particular resource is at risk and for what reasons.
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:32]: @drichards-87 you can remove this line completely. So, you can take out the "Relationships" tab screenshot and add the screenshot for "Resource" configuration tab.

File: content/en/security/identity_risks/_index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Review and remediate identity risk detections
+
+Review your organization's active identity risk detections on the [Identity Risks Explorer page][3]. Use the **Group by** options to filter by **Identity Risks**, **Resources**, or **None** (individual detections). View additional details on the side panel by selecting a resource or detection.
+
+{{< img src="security/identity_risks/identity_risks_explorer.png" alt="CSM Identity Risks explorers page" width="100%">}}
+
+When viewing an individual resource, use the **Relationships** tab to view a graphical representation of the connections with other resources. The view includes both benign and at-risk connections and shows how a particular resource is at risk and for what reasons. 
+
+You can also review the configuration of the resource on the **Resource** tab, and a chronological history of the identity risks detected for the resource on the **Timeline** tab.
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:34]: You can review the configuration of the affected resource on the **Resource** tab.

@drichards-87 I took out the comment about timeline as we might remove that tab.

File: content/en/security/identity_risks/_index.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+{{< img src="security/identity_risks/relationships_tab_side_panel.png" alt="The Relationships tab shows a a graphical representation of the connections with other resources" width="80%">}}
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:36]: This image can be removed. We need to add the screenshot for the "Resource" configuration tab.

File: content/en/security/identity_risks/_index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+On the side panel for an individual detection, click **Fix in AWS** to open the AWS console to remediate the identity risk. You can also use the **Insights** tab to view the permissions provisioned on the resource and whether they were used in the last 15 days.
+
+{{< img src="security/identity_risks/insights_tab_side_panel.png" alt="The Insights tab shows a list of permissions provisioned on the resource" width="80%">}}
+
+## Further Reading
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:42]: Let's add the Datadog CIEM announcement blog here in the "Further Reading" section when ready from @mallorym.

File: content/en/security/cloud_security_management/_index.md [hunk 9]
```diff
@@ -35,13 +35,12 @@ further_reading:
...
+Datadog Cloud Security Management (CSM) delivers real-time threat detection and continuous configuration audits across your entire cloud infrastructure - in a unified view for seamless collaboration and faster remediation. Powered by observability data, security teams can determine the impact of a threat quickly by tracing the full attack flow and identify the resource owner where a vulnerability was triggered. Engineers are able to actively monitor their security risks by incorporating security metrics into their existing workflow.
 
-Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#cloud-security-posture-management), and [Cloud Workload Security (CWS)](#cloud-workload-security).
+Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#cloud-security-posture-management), [Cloud Workload Security (CWS)](#cloud-workload-security), and [CSM Identity Risks](#csm-identity-risks).
```
Comment (rajat-luthra-ddog) [inline content/en/security/cloud_security_management/_index.md:40]: Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#cloud-security-posture-management), [Cloud Workload Security (CWS)](#cloud-workload-security), [Cloud Infrastructure Entitlement Management (CIEM)](#csm-identity-risks), and [Infrastructure Vulnerability Management](link per @aliciascott).

File: content/en/security/cloud_security_management/_index.md [hunk 10]
```diff
@@ -54,6 +53,12 @@ Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#c
 
 {{< img src="security/cws_overview.png" alt="Cloud Workload Security coverage views in Datadog" width="100%">}}
 
+## CSM Identity Risks
```
Comment (rajat-luthra-ddog) [inline content/en/security/cloud_security_management/_index.md:56]: Cloud Infrastructure Entitlement Management

File: content/en/security/cloud_security_management/_index.md [hunk 11]
```diff
@@ -54,6 +53,12 @@ Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#c
...
+
+[CSM Identity Risks][4] provides in-depth visibility into your organization's AWS IAM risks. Enabling you to detect and resolve identity risks on an ongoing basis, CSM Identity Risks helps secure your cloud infrastructure from IAM-based attacks.
```
Comment (rajat-luthra-ddog) [inline content/en/security/cloud_security_management/_index.md:58]: [Cloud Infrastructure Entitlement Management (CIEM)][4], surfaced in CSM (hyperlink) as "Identity Risks", provides in-depth visibility into your organization's AWS IAM risks. It enables you to detect and resolve identity risks on an ongoing basis to secure your cloud infrastructure from IAM-based attacks.

File: content/en/security/cloud_security_management/_index.md [hunk 12]
```diff
@@ -63,3 +68,4 @@ To get started with Datadog Security, navigate to the [Setup & Configuration][3]
 [1]: /security/cspm/
 [2]: /security/cloud_workload_security/
 [3]: https://app.datadoghq.com/security/configuration
+[4]: /security/identity_risks
```
Comment (rajat-luthra-ddog) [inline content/en/security/cloud_security_management/_index.md:71]: /security/ciem

**Review by rajat-luthra-ddog — CHANGES_REQUESTED** [review]
Added all change suggestions as comments for both pages - CSM and CIEM. Thanks

File: content/en/security/identity_risks/_index.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
+---
+title: Cloud Security Management Identity Risks
```
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/security/identity_risks/_index.md:2]: I can definitely do that, but just want to call out that we're changing CSPM and CWS to be CSM Misconfigurations and CSM Threats. Those changes are in a separate PR that will be merged next week.

File: content/en/security/cloud_security_management/_index.md (hunk 9 above)
Comment (drichards-87) [reply content/en/security/cloud_security_management/_index.md:40]: The change you mention will be in a separate PR (Alicia's). We will also be completely revamping this page as part of the CSM rebrand (also a separate PR).

File: content/en/security/identity_risks/_index.md (hunk 8 above)
Comment (drichards-87) [reply content/en/security/identity_risks/_index.md:42]: Sounds good @rajat-luthra-ddog. @mallorym please feel free to submit a PR once the announcement blog is posted.

File: content/en/security/cloud_security_management/_index.md (hunk 9 above)
Comment (rajat-luthra-ddog) [reply content/en/security/cloud_security_management/_index.md:40]: Ok. Thanks.

File: content/en/security/identity_risks/_index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (rajat-luthra-ddog) [reply content/en/security/identity_risks/_index.md:2]: Ok. Thanks.

File: content/en/security/identity_risks/_index.md (hunk 2 above)
Comment (rajat-luthra-ddog) [reply content/en/security/identity_risks/_index.md:20]: @drichards-87 can we please make this change? Ending a sentence with the word 'only' isn't ideal.

File: content/en/security/identity_risks/_index.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+CSM Identity Risks is not available in this site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">CSM Identity Risks is in beta.</div>
+
+Cloud Security Management Identity Risks (CSM Identity Risks) provides in-depth visibility into your organization's AWS IAM risks. It enables you to detect and resolve identity risks on an ongoing basis to secure your cloud infrastructure from IAM-based attacks.
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:18]: Let's remove AWS from the above line ("... into your organization's IAM risks.") as we are clarifying the current limitation (AWS only) below?

File: content/en/security/identity_risks/_index.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+<div class="alert alert-info">At this time, CSM Identity Risks is available for AWS only.</div>
+
+## Setup
+
+To use CSM Identity Risks, you must [enable resource collection for AWS][2] and [enable CloudTrail logs forwarding][4]. If you've already done this, no additional setup is required.
```
Comment (rajat-luthra-ddog) [inline content/en/security/identity_risks/_index.md:24]: Can you point the [2] link above to https://docs-staging.datadoghq.com/deforest/csm-identities/integrations/amazon_web_services/?tab=roledelegation#cloud-security-posture-management?

**Review by rajat-luthra-ddog — COMMENTED** [review]
Submitting changes.

**Pull request merged**